esm-samples/jsapi-node/README.md

```diff
@@ -1,48 +1,47 @@
-# ArcGIS API for JavaScript in Node
+# ArcGIS API for JavaScript in Node.js
 
-Integrating Node with [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) can be done by building the app with [native ESM in a supported Node version](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html) or by transpiling to CommonJS.
+Integrating Node.js with [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) can be done by building the app with [native ESM in a supported Node version](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html) or by transpiling to CommonJS. This sample contains examples of both approaches.
+
+## Working with assets
+
+Working with the ArcGIS API for JavaScript's ES modules in Node.js requires configuring the APIs assets, since the application will be running on a local server and not in a web app. The assets include styles, images, fonts, and localization files. Be sure to set [`config.assetsPath`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath) so that the assets are correctly resolved, for example:
 
 ```js
-// rollup.config.js
-{
-import commonjs from "@rollup/plugin-commonjs";
-import resolve from "@rollup/plugin-node-resolve";
-
-export default {
-  input: ["src/request.js", "src/projection.js", "src/webmap.js"],
-  output: {
-    chunkFileNames: "chunks/[name].js",
-    dir: "dist",
-    format: "cjs"
-  },
-  external: ["whatwg-fetch"],
-  plugins: [resolve(), commonjs()],
-  preserveEntrySignatures: false
-};
+import esriConfig from '@arcgis/core/config.js';
+esriConfig.assetsPath = "./public/assets"; // relative to when running in root
```

Comment:
The assets don't need to be copied, so this should use what it was using below:

`esriConfig.assetsPath = "node_modules/@arcgis/core/assets"; // relative to when running in root`

Reply:
Fixed.

Reply:
Fixed.